### PR TITLE
Elixir 1.10 + hexpm docker image

### PR DESCRIPTION
**Pull request by gcauchon**
Elixir `1.10` was released on January 27th with [a few nice enhancements](https://elixir-lang.org/blog/2020/01/27/elixir-v1-10-0-released/)!

In a related effort, the elixir/hex core team are [now building Docker images](https://elixirforum.com/t/yet-another-elixir-and-erlang-docker-image/28740) that are Elixir+Erlang+Alpine specific, and way more stable thant the "official" ones.

### CODE REVIEW

File: assets/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "private": true,
   "engine-strict": true,
   "engines": {
-    "node": "^10.16",
+    "node": "^12.14",
```
Comment (Boubalou) [inline assets/package.json]: Shall we go straight to 12.15+ if we are to upgrade it?

Comment (gcauchon) [reply assets/package.json]: `12.14` was the lastest LTS when I started this PR… I’ll see if it’s possible to upgrade to `12.15`

Comment (Boubalou) [reply assets/package.json]: Because of this: https://snyk.io/blog/node-js-release-fixes-a-critical-http-security-vulnerability/

Comment (gcauchon) [reply assets/package.json]: Thanks, ✅

File: priv/scripts/ci-check.sh [hunk 2]
```diff
@@ -30,27 +30,18 @@ run() {
 header "Lint files…"
 run make lint
 
-header "Check code format…"
-run make check-format
+header "Check codebase…"
+run make check
```
Comment (Boubalou) [inline priv/scripts/ci-check.sh]: I didn't see this in the `Makefile`. How does it work?

Also, I prefer having them in separate calls with a header adding details on what task in running. What's the current output of the `make check` task?

Comment (gcauchon) [reply priv/scripts/ci-check.sh]: I made this  change for consistency with the `lint` target that wraps `lint-elixir lint-scripts lint-styles` in a single `make` command.

The output looks like this:

```
> make check
mix format --dry-run --check-formatted
./assets/node_modules/.bin/prettier --check 'assets/.babelrc' 'assets/webpack.config.js' 'assets/{js,css,scripts}/**/*.{js,graphql,scss,css}' '**/*.md'
Checking formatting...
All matched files use Prettier code style!
mix sobelow --config
##############################################
#                                            #
#          Running Sobelow - v0.10.0         #
#  Created by Griffin Byatt - @griffinbyatt  #
#     NCC Group - https://nccgroup.trust     #
#                                            #
##############################################

... SCAN COMPLETE ...

mix coveralls

15:20:04.161 [info]  Already up
.........

Finished in 0.3 seconds
9 tests, 0 failures

Randomized with seed 625404
----------------
COV    FILE                                        LINES RELEVANT   MISSED
100.0% lib/elixir_boilerplate.ex                       9        0        0
 80.0% lib/elixir_boilerplate/application.ex          24        5        1
100.0% lib/elixir_boilerplate/gettext.ex               8        0        0
  0.0% lib/elixir_boilerplate/release_tasks.ex        22        6        6
100.0% lib/elixir_boilerplate/repo.ex                 13        0        0
  0.0% lib/elixir_boilerplate_graphql/applicati       17        1        1
100.0% lib/elixir_boilerplate_graphql/compiled_        9        0        0
100.0% lib/elixir_boilerplate_graphql/elixir_bo        5        0        0
 66.7% lib/elixir_boilerplate_graphql/plugs/con       11        3        1
 33.3% lib/elixir_boilerplate_graphql/router.ex       26        3        2
  0.0% lib/elixir_boilerplate_graphql/schema.ex       27        2        2
100.0% lib/elixir_boilerplate_health/elixir_boi       13        0        0
100.0% lib/elixir_boilerplate_web/content_secur       43       12        0
 76.0% lib/elixir_boilerplate_web/endpoint.ex        141       25        6
100.0% lib/elixir_boilerplate_web/errors/errors       59       13        0
100.0% lib/elixir_boilerplate_web/errors/view.e        7        1        0
100.0% lib/elixir_boilerplate_web/home/controll       10        1        0
100.0% lib/elixir_boilerplate_web/home/view.ex         3        0        0
100.0% lib/elixir_boilerplate_web/layouts/view.        8        0        0
 75.0% lib/elixir_boilerplate_web/router.ex           34        4        1
100.0% lib/elixir_boilerplate_web/socket.ex            9        0        0
[TOTAL]  73.7%
----------------
mix check.unused_locked_deps
```

Comment (Boubalou) [reply priv/scripts/ci-check.sh]: Excellent then! I guess that I really missed the change to the Makefile on my first read. 🤷‍♂

**Pull request merged**